pig/src/main/resources/interpreter-setting.json

```diff
@@ -15,6 +15,12 @@
         "propertyName": "zeppelin.pig.includeJobStats",
         "defaultValue": "false",
         "description": "flag to include job stats in output"
+      },
+      "zeppelin.pig.execType": {
+        "envName": "SPARK_MASTER",
+        "propertyName": "SPARK_MASTER",
```

Comment:
```
if ZEPPELIN_SPARK_CONCURRENTSQL is set but property zeppelin.spark.concurrentSQL is not, then it will be used as zeppelin.spark.concurrentSQL
```
Unfortunately I think this is an issue of zeppelin itself. The fallback mechanism only take effect in `ZeppelinConfiguration`. But Zeppelin use `java.util.Properties` in Interpreter, so only propertyName would be used in Interpreter.

If the property user defined in interpreter setting is all capitalized, then it would passed to interpreter process as env variable, otherwise it would be passed as property. And fallback mechanisms won't take effect, it is interpreter's responsibility to choose to use env or property. This is a bad design of zeppelin, unfortunately it would not be fixed in short time.